common/src/main/java/org/opensearch/ml/common/MLModel.java

```diff
@@ -144,4 +170,8 @@ public static MLModel fromStream(StreamInput in) throws IOException {
         MLModel mlModel = new MLModel(in);
         return mlModel;
     }
+
+    public static String customModelId(String modelName, Integer version, Integer chunkNumber) {
+        return modelName + "_" + version + "_" + chunkNumber;
```

Comment:
Does this ensure the return modelId is unique? Just wondering if this would trigger problems in model retrieving for models with "null" chunkNumber.

Reply:
Just this method itself won't ensure the model id is unique. The custom model uploading makes sure the chunkNumber is not null.